README.md

```diff
@@ -1 +1,94 @@
-# oms-api
\ No newline at end of file
+# oms-api
+
+# Prerequisite
+Ionic CLI - If you don't have the ionic CLI installed refer [official documentation](https://ionicframework.com/docs/intro/cli) for the installation instructions.
+
+
+# Build Notes (Users)
+
+1. Download the app from [release](https://github.com/hotwax/oms-api/releases) page and extract it.
+2. Go to the app directory.
+3. Run following command to download dependencies  
+    `npm i
+4. Create a `.env` file by taking reference from the `.env.example`.
+5. To run the app in browser use the command: `ionic serve`
+
+
+# Build Notes (Contributors)
+
+1. Open a Terminal window
+2. Clone app using the command: `git clone https://github.com/hotwax/oms-api.git <repository-name>`
+3. Go to the <repository-name> directory using command: `cd <repository-name>`
+4. Run following command to download dependencies
+    `npm i`
+5. Create a `.env` file by taking reference from the `.env.example`.
+6. To run the app in browser use the command: `ionic serve`
+
+# Firebase Hosting
```

Comment:
Remove this heading, sub-headings and it's content.

Reply:
Okay I have removed it.